src/app/pages/StoryPage/fixtureData/mundo.json

```diff
@@ -2131,5 +2131,337 @@
     },
     "id": "urn:bbc:ares::asset:mundo/23263889",
     "type": "cps"
+  },
+  "secondaryColumn": {
```

Comment:
Wasn't immediately obvious to me but this fixture data is added to facilitate the new storybook stories added to test the layout with various data missing.

Reply:
(just leaving this comment to help others, not suggesting any change)